composer.json

```diff
@@ -22,9 +22,9 @@
     "php": "^8.0",
```

Comment:
This might be unrelated to this PR, but PHP 8.0 is the minimum version required by Laravel 9, which is deprecated. Shouldn't we consider bumping this to PHP 8.1 and setting our minimum Laravel version to Laravel 10, which is still supported?

https://laravel.com/docs/11.x/releases

Reply:
Appreciate the response mamosta.